ui/app/templates/rules.html

```diff
@@ -302,7 +302,7 @@ <h5 title="Fallback Mapping" ng-show="rule.fallbackMapping || (show_sc && rule.s
         <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="buildID" fieldtitle="Build ID"></rulefield>
         <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="buildTarget" fieldtitle="Build Target"></rulefield>
         <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="locale" fieldtitle="Locale(s)"></rulefield>
-        <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="distribution" fieldtitle="Distribution"></rulefield>
+        <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="distribution" fieldtitle="Distribution" limittextto="20" help="{{'Distribution list: ' + rule.distribution}}"></rulefield>
```

Comment:
Is it possible to have the entire string shown as a tooltip? Right now, the only way to see it is to open a dialog, which isn't terribly convenient. I think you should do this on rule_scheduled_changes.html, too.